src/renderer/components/prompts/folder.jsx

```diff
@@ -23,25 +22,21 @@ export default React.createClass({
 
   render: function () {
 
-    const raisedButtonStyle = {
-      margin: '56px 24px 0px 24px'
-    };
-
     const isEmpty = !!this.props.selectedFolder;
 
     return (
-      <div className="select-folder-prompt fieldset">
-        <label style={{ background: this.props.color }}>
+      <div style={this.getContainerStyle()}>
+        <label style={this.getLabelStyle()}>
```

Comment:
This is just a preference, but I would much rather see a `<Label />` component than a mixin that adds functions to the component to get styles when that function always returns the same styles.  (This goes for Container as well)